### DISCUSSION TITLE
When Will Bromite Start Supporting Desktop

### DISCUSSION THREAD

**Discussion by spiderman973** [bromite/bromite#1389] [Q&A] [answered]
When Will Bromite Supoort Desktop I wanna use Bromite on my De-Microsofted Windows Laptop Bromite is awesome!

**Comment by chromer030** [comment]
Bromite for Desktop will be something like ungoogled-chromium , you can use it , most of the patches of Bromite come from ungoogled-chromium project.

**Reply by spiderman973** [reply]
But Disabling JIT compiler in Ungoogled-Chromium is not possible

**Reply by chromer030** [reply]
maybe using extension for this purpose ?

**Reply by csagan5** [reply]
> most of the patches of Bromite come from ungoogled-chromium project.

That is false; you can check yourself.

**Reply by ghost** [reply]
> But Disabling JIT compiler in Ungoogled-Chromium is not possible

i also said JIT in other discussion and issue @chromer030 @csagan5  misunderstood as Javascript not JIT

**Comment by csagan5** [comment, marked as answer]
There are currently no plans to provide such builds.

**Reply by Alex313031** [reply]
Hello, I am creating a fork for linux, and I would like to include the bromite logo, but wish to edit it to be black, blacker, blackest, rather than green, greener, greenest. Anyway, I'm asking permission even though its GPL3, but also the dirs related to logo change seem to be missing from your repo. Where can I get this, or if needed I'll do it myself, but I would like the highest res, or uncompressed/lossless, or 8x8...16x16...32x32...etc. icon set and any related UI changes/icon changes within bromite.

As needed changes would be made to build properly  on linux and I will be including widevine and all thinlto and pgo optimizations.
@spiderman973 @csagan5

**Comment by Christopher-Chros** [comment]
Ik this was already answered but I just wanted to say, Bromite would be a great alternative to something like Brave, by keeping the vanilla functions of chromium mixed with ad-blocking and tracker-blocking. And unlike Brave, none of the shady business practices. I can see Bromite becoming a major privacy-based alternative to Chromium (ik Bromite is based off of Chromium), especially with all the talks about Firefox losing users and possibly not existing in the near future, it would be refreshing to see a new privacy-based browser taking the charge.

**Reply by Alex313031** [reply]
Why would it not exist in the near future.

**Reply by Universalizer** [reply]
> Why would it not exist in the near future.

My neutral (natural facts) comments on Iceraven Browser for Android (innocent FOSS Project),
Read carefully and understand.

Especially first link
https://github.com/fork-maintainers/iceraven-browser/issues/26#issuecomment-1345247959

https://github.com/fork-maintainers/iceraven-browser/issues/26#issuecomment-1345366115
